docker/README.md

```diff
@@ -22,6 +22,20 @@ nginx. To build, run something like
 $ docker build -t openzipkin/zipkin-ui:test -f docker/Dockerfile --target zipkin-ui .
 ```
 
+## zipkin-kafka Docker image
```

Comment:
Can you either move the Cassandra readme here or this to the subdirectory? Either is fine if consistent

Reply:
done